integration_tests/seeds/dag/dag_seeds.yml

```diff
@@ -73,4 +73,13 @@ seeds:
             - parent
             - child
             - parent_and_child
-            - is_loop_independent
\ No newline at end of file
+            - is_loop_independent
+
+  - name: test_fct_hard_coded_references
+    tests:
+      - dbt_utils.equality:
+          name: equality_fct_hard_coded_references
+          compare_model: ref('fct_hard_coded_references')
+          compare_columns:
```

Comment:
nit: since the compare columns are all the columns, we can just leave out this config